HW2/P2/P2.py

```diff
@@ -10,6 +10,8 @@
 import numpy as np
 from timer import Timer
 from parallel_vector import move_data_serial, move_data_fine_grained, move_data_medium_grained
+import matplotlib.pyplot as plt
```

Comment:
Overall, well done! You did better than me, frankly!  Commented very well. Sorry, I thought I peer reviewed yours, but it turned out I didn't.